ISTIO.md

```diff
@@ -0,0 +1,74 @@
+# Istio manifests for test
+
+This repository is testing two versions of [Istio](https://istio.io/) - `isito-head` and `istio-latest`.
+
+- [`istio-head`](third_party/istio-head/) downloads the nightly Istio build for test.
+- [`istio-latest`](third_party/istio-latest/) downloads the latest Istio release for test.
+
+The download scripts is hosted under [`third_party`](third_party) directory.
+
+## Bump Istio manifests
+
+Currently the manifest files of Istio under the [`third_party`](third_party) directory must be updated manually when Istio released the latest version.
+
+Before bumping Istio version, you must install these tools:
+
+- [`yq (v4)`](https://github.com/mikefarah/yq): For tweak manifest files.
+
+### Bump istio-latest manifests
+
+The following steps show how you can bump the `istio-latest` manifest:
+
+#### 1. Set the version number to bump
+
+  The version number is defined in `third_party/istio-latest/generate-manifests.sh`. For example, if you want to bump the Istio version to `1.13.2`,
+  you can edit `third_party/istio-latest/generate-manifests.sh` and set the version as:
+
+  ```sh
+  generate "1.13.2" "$(dirname $0)"
+  ```
+
+#### 2. Run `hack/update-codegen.sh`
+
+  The script updates manifest automatically so you just kick it:
+
+  ```sh
+  ./hack/update-codegen.sh
+  ```
+
+#### 3. Send the PR
+
+  Once the manifest was updated, please send the PR against net-istio repository. The CI starts using the updated manifest and detect any issue if exists.
```

Comment:
Recommend changing `manifest was updated` to `manifest is updated`.

Also recommend changing `detect` to `detects`